### DISCUSSION TITLE
Is there a way to move accordion arrow to the left instead of right

### DISCUSSION THREAD

**Discussion by BilalAtique** [saadeghi/daisyui#2284] [Q&A] [answered]
![image](https://github.com/saadeghi/daisyui/assets/74999164/d58e1cff-5fbb-41e1-ad97-b9cb7d2efe0a)
I want to move the accordion arrow to the left instead of the right. Is there any prop for it? If not, is there any hacky way to achieve this?

**Comment by saadeghi** [comment, marked as answer]
You can do that by adding `after:left-[1.4rem] pl-12 pr-4` class names to `collapse-title`  
Either in HTML: https://play.tailwindcss.com/Tg2O7wSJmu
Or in CSS: https://play.tailwindcss.com/LDNdrP0qLQ?file=css
